### ISSUE TITLE
Online demo would be cool

### ISSUE THREAD

**Issue by JuhQ** [philcockfield/storybook-host#36]

**Comment by philcockfield** [comment]
Good idea - I've got some cool stuff in the pipeline for this, so should get one going.

Out of interest, wow did you come across it?

**Comment by JuhQ** [comment]
We're using storybook in our project and I'm constantly on the look for new plugins to improve our workflow and tooling.

I came across the project from the new github dashboard, under the `Discover repositories` panel :)
